src/main/java/seedu/address/model/CommandRecord.java

```diff
@@ -1,21 +1,19 @@
 package seedu.address.model;
 
-import seedu.address.commons.exceptions.AlfredModelHistoryException;
 import seedu.address.ui.EntityCard;
 
 /**
- * Represents a Previous, Current or Future Command in History.
+ * Represents a record of a previously executed command in ModelHistory.
  */
 public class CommandRecord {
     /**
      * Represents the CommandType of a CommandRecord.
-     * As the user uses the undo or redo command,
-     * the commandtype will represent whether it
-     * is a future command for redo, past command for undo,
-     * end command so that user can no longer undo,
-     * or current command user is at.
+     * Indicates whether the CommandRecord represents a Command
+     * that can be undone (UNDO), redone (REDO), is the current
+     * command (CURR), or is a sentinel for the endpoints (END).
+     * The user will not be able to undo/redo beyond the endpoints.
      */
-    public enum CommandType { PAST, CURR, FUTURE, END };
+    public enum CommandType { UNDO, CURR, REDO, END };
```

Comment:
I presume this means the commands that can be undone or redo?

Reply:
Yes! I changed the values from PAST to UNDO and FUTURE to REDO because all the commands are technically in the past, as they have all been previously executed (that's why they are in ModelHistory). Hope you don't mind this simple relabelling. :)